docs/components/plugins/APISection.tsx

```diff
@@ -35,7 +35,14 @@ const renderAPI = (
   try {
     const data = require(`~/public/static/data/${version}/${packageName}.json`).children;
 
-    const methods = filterDataByKind(data, TypeDocKind.Function);
+    const methods = filterDataByKind(
+      data,
+      TypeDocKind.Function,
+      entry => !entry.name.includes('Listener')
+    );
+    const eventSubscriptions = filterDataByKind(data, TypeDocKind.Function, entry =>
+      entry.name.includes('Listener')
+    );
```

Comment:
```suggestion
    const eventSubscriptions = filterDataByKind(
      data,
      TypeDocKind.Function,
      entry => entry.name.includes('Listener')
    );
```
I'm not a `code-style-nazi`, but I prefer the consequent code lines to look the same if these are almost similar in terms of arguments number & length 😉

Reply:
Generally code is formatted by ESLint (and to be more specific Prettier), I will try to change it to the suggested format, but it might be reverted by the `lint` command. 😉

Reply:
@bbarthec Unfortunately, as I was expecting, the ESLint check has failed:
<img width="840" alt="Screenshot 2021-04-13 114227" src="https://user-images.githubusercontent.com/719641/114532768-75efd980-9c4d-11eb-88c4-033bc5615ceb.png">

Reply:
😮 Oh, strange formatting, but I guess we can live with it then 😉